core/keyboard_nav/cursor_svg.js

```diff
@@ -26,25 +26,20 @@
 
 goog.provide('Blockly.CursorSvg');
 
```

Comment:
2 spaces between the provide / requires and start of the file.

Reply:
done